Dockerfile

```diff
@@ -167,6 +167,7 @@ RUN set -ex && \
 RUN set -ex && \
     pip3 install --no-index --find-links=./ /build_output/wheels/*.whl && \
     cp /build_output/deps/skopeo /usr/bin/skopeo && \
+    curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /anchore_engine/bin v0.3.0 && \
```

Comment:
This needs to happen in the first stage. The binary should be installed to the `/build_output` directory, then copied to a directory in the PATH in the second stage. We use the tarball created from the /build_output directory to build our DCAR(DOD) images.